docs/specification.md

```diff
@@ -340,29 +344,23 @@ Here is a snippet of sample Go code to illustrate:
     }
   }
 ```
+Here is a snippet of sample Python code to illustrate using `grpc-retry-pushback-ms`:
 
-When the client receives this signal, it SHOULD follow the recommendations
-outlined in documentation for
-[RetryInfo](https://github.com/googleapis/googleapis/blob/6a8c7914d1b79bd832b5157a09a9332e8cbd16d4/google/rpc/error_details.proto#L40):
-
-```
-// Describes when the clients can retry a failed request. Clients could ignore
-// the recommendation here or retry when this information is missing from the error
-// responses.
-//
-// It's always recommended that clients should use exponential backoff when
-// retrying.
-//
-// Clients should wait until `retry_delay` amount of time has passed since
-// receiving the error response before retrying.  If retrying requests also
-// fail, clients should use an exponential backoff scheme to increase gradually
-// the delay between retries based on `retry_delay` until either a maximum
-// number of retries has been reached, or a maximum retry delay cap has been
-// reached.
+```go
+# Do this on the server side.
+trailer := metadata.Pairs("grpc-retry-pushback-ms", "5000")
+grpc.SetTrailer(ctx, trailer)
 ```
 
-The value of `retry_delay` is determined by the server and is implementation
-dependant. The server SHOULD choose a `retry_delay` value that is big enough to
+On the client side using gRPC retry config (https://grpc.io/docs/guides/retry/) 
+will cause gRPC to automatically parse this header and handle all backoff and retry logic.
+
+When the client receives this signal, if it isn't using gRPC retry config, it SHOULD 
+follow the recommendations or outlined in documentation for
+[RetryInfo](https://github.com/googleapis/googleapis/blob/6a8c7914d1b79bd832b5157a09a9332e8cbd16d4/google/rpc/error_details.proto#L40).
```

Comment:
> if it isn't using gRPC retry config

This is not clear for me.
Does it suggest that the client should support only one retry pushback mechanism?
Would it be bad to support both of them?

How about

```suggestion
The description of how the client should respect `RetryInfo` is provided
in its [documentation](https://github.com/googleapis/googleapis/blob/6a8c7914d1b79bd832b5157a09a9332e8cbd16d4/google/rpc/error_details.proto#L34-L39).
```